src/main/java/org/schemaspy/model/Catalog.java

```diff
@@ -19,25 +19,22 @@
 package org.schemaspy.model;
 
 import java.util.Map;
+import java.util.Objects;
 
 import org.schemaspy.util.CaseInsensitiveMap;
 
 public final class Catalog implements Comparable<Catalog>{
     private String name;
```

Comment:
Consider making the field `name` final and remove the setter as it is never used (as far as I can see).